src/config/index.js

```diff
@@ -0,0 +1,7 @@
+const { ASTEROIDS_FEED_URL, API_KEY, PORT } = require('./environment');
```

Comment:
environment.js and index.js are almost identical, looks little bit weird

Reply:
index.js file removed.